index.js

```diff
@@ -15,9 +15,18 @@ module.exports = async function serve (drive, opts = {}) {
       return
     }
 
-    const { pathname } = new URL(req.url, 'http://localhost')
+    const { pathname, searchParams } = new URL(req.url, 'http://localhost')
+    const version = searchParams.get('checkout')
+    const driveForReq = version ? drive.checkout(version) : drive
```

Comment:
`snapshot` instead of `driveForReq`? I like naming things as they're exemplified in their own README like hyperdrive-next, etc